js2-mode.el

```diff
@@ -10787,6 +10789,16 @@ When `js2-is-in-destructuring' is t, forms like {a, b, c} will be permitted."
               ;; Unquoted names: {foo: 12}
               ((= tt js2-NAME)
                (js2-create-name-node))
+              ;; Rest/spread property: {...foo}
+              ((and (>= js2-language-version 200)
+                    (= tt js2-TRIPLEDOT)
+                    ;; FIXME: In destructuring ("rest") it must be just a name,
+                    ;; but outside of destructuring ("spread") it can be any
+                    ;; expression.
```

Comment:
AssignmentExpression, per spec. (But most of the use in my codebase so far just uses simple names.)